### ISSUE TITLE
SwinUNETR on BTCV with unused CLI arguments

### ISSUE THREAD

**Issue by razorx89** [Project-MONAI/research-contributions#63]
**Describe the bug**
The training script for SwinUNETR on BTCV includes CLI arguments for smoothing coefficients in the Dice loss. However, those coefficients are never used. Were they used for training the original models reported in the publication?

https://github.com/Project-MONAI/research-contributions/blob/46279a9601852a1e5f3a2581fa32be403f57885e/SwinUNETR/BTCV/main.py#L78-L79

https://github.com/Project-MONAI/research-contributions/blob/46279a9601852a1e5f3a2581fa32be403f57885e/SwinUNETR/BTCV/main.py#L144-L146

@ahatamiz

**Comment by ahatamiz** [comment]
Hi @razorx89 

Thanks for the comment. Those coefficients particularly were intended to be used for squared Dice loss. However, we did not use it for training the models whose benchmarks are reported in the papers. I have submitted a PR with the option of using squared Dice loss, in case you would like to give it a try. 

Best,

**Comment by razorx89** [comment]
Thanks.
